app/models/country.rb

```diff
@@ -4,4 +4,62 @@ class Country < ApplicationRecord
+    natural_resources = (defined? json[:Geography][:"Natural resources"][:text]) ? json[:Geography][:"Natural resources"][:text] : "No Data"
+    environment = (defined? json[:Geography][:Terrain][:text]) ? json[:Geography][:Terrain][:text] : "No Data"
+    ethnic_breakdown = (defined? json[:"People and Society"][:"Ethnic groups"][:text]) ? json[:"People and Society"][:"Ethnic groups"][:text] : "No Data"
+    population = (defined? json[:"People and Society"][:Population][:text]) ? json[:"People and Society"][:Population][:text] : "No Data"
+    languages = (defined? json[:"People and Society"][:Languages][:text]) ? json[:"People and Society"][:Languages][:text] : "No Data"
+    religions = (defined? json[:"People and Society"][:Religions][:text]) ? json[:"People and Society"][:Religions][:text] : "No Data"
+    age_structure = (defined? json[:"People and Society"][:"Age structure"]) ? json[:"People and Society"][:"Age structure"] : "No Data"
+    median_age = (defined? json[:"People and Society"][:"Median age"][:total][:text]) ? json[:"People and Society"][:"Median age"][:total][:text] : "No Data"
+    net_migration_rate = (defined? json[:"People and Society"][:"Net migration rate"][:text]) ? json[:"People and Society"][:"Net migration rate"][:text] : "No Data"
+    urbanization = (defined? json[:"People and Society"][:Urbanization][:"urban population"][:text]) ? json[:"People and Society"][:Urbanization][:"urban population"][:text] : "No Data"
+    sex_ratio = (defined? json[:"People and Society"][:"Sex ratio"][:"total population"][:text]) ? json[:"People and Society"][:"Sex ratio"][:"total population"][:text] : "No Data"
+    capital = (defined? json[:Government][:Capital][:name][:text]) ? json[:Government][:Capital][:name][:text] : "No Data"
+    dual_citizentship = (defined? json[:Government][:Citizenship][:"dual citizenship recognized"][:text]) ? json[:Government][:Citizenship][:"dual citizenship recognized"][:text] : "No Data"
+    residency_requirement = (defined? json[:Government][:Citizenship][:"residency requirement for naturalization"][:text]) ? json[:Government][:Citizenship][:"residency requirement for naturalization"][:text] : "No Data"
+    government_type = (defined? json[:Government][:"Government type"][:text]) ? json[:Government][:"Government type"][:text] : "No Data"
+    gdp_per_capita = (defined? json[:Economy][:"GDP - per capita (PPP)"][:text]) ? json[:Economy][:"GDP - per capita (PPP)"][:text] : "No Data"
+    unemployment_rate = (defined? json[:Economy][:"Unemployment rate"][:text]) ? json[:Economy][:"Unemployment rate"][:text] : "No Data"
+    population_below_poverty_line = (defined? json[:Economy][:"Population below poverty line"][:text]) ? json[:Economy][:"Population below poverty line"][:text] : "No Data"
+    exports = (defined? json[:Economy][:"Exports - commodities"][:text]) ? json[:Economy][:"Exports - commodities"][:text] : "No data"
+
+
+
+    Country.find_by(name: name).update_attributes!(
```

Comment:
@vidoseaver Are you sure you want to raise an exception here (the `!`)?

Seeing the `puts` statement below, I'm going to guess that there's an import that gets run possibly from a rake task. If that's the case, let's break this out into a CountryImporter PORO or something like that.

This is a long method but I don't know if it's avoidable. Importing tends to be kinda gross.